blocks/footer/footer.css

```diff
@@ -1,14 +1,267 @@
+  border-radius: 3.56rem;
+  box-shadow: 0 1.25rem 2.5rem 0 #0e1f354d;
+  padding: 1.25rem 3.125rem; ;
+  width: 100%;
+  margin-bottom: -3.75rem !important;
+}
+
+.section .footer-top-container {
+  padding-top: 0;
+}
+
+.footer-top p {
+  font-size: 0.9375rem;
+  margin-bottom: 0.5rem !important;
+}
+
+.footer-top .col-1  img{
+    flex: 0 0 auto;
+    border: 0;
+    vertical-align: middle;
+    max-width: 100%;
+    width: 17.19rem;
+    height: 2.375rem;
+}
+
+.footer-top .col-2 {
+  flex: 0 0 auto;
+  width: 50%;
+  text-align: center;
+  position: relative;
+  margin-bottom: 0;
+}
+
+.footer-top .col-2::before {
+  height: 6rem;
+  background: #9797974d;
+  width: 0.0625rem;
+  content: "";
+  position: absolute;
+  left: 1.25rem;
+  top: 0;
+  bottom: 0;
   margin: auto;
 }
 
-footer .footer p {
-  margin: 0;
+.footer-top .col-2::after {
+  height: 6rem;
+  background: #9797974d;
+  width: 0.0625rem;
+  content: "";
+  position: absolute;
+  right: 1.25rem;
+  top: 0;
+  bottom: 0;
+  margin: auto;
+}
+
+.footer-disclaimer p {
+  color: #a3a3a3;
+  font-size: 0.6875rem;
+  line-height: 0.8125rem;
+}
+
+.footer-top .col-3 {
+  flex: 0 0 auto;
+  width: 25%;
+}
+
+.footer-top .col-3 p {
+  text-align: left;
+  padding-left: 1rem;
+}
+
+.footer-links {
+  background: #3d3d3d;
+  padding: 6.875rem 0 0;
+} 
+
+.footer-links .links-title {
+  font-size: 1rem;
+  line-height: 130%;
+  color: var(--color-white);
+  font-weight: 500;
+  display: block;
+  margin-bottom: 0.75rem;
+}
+
+.footer-links a {
+  color: #a3a3a3;
+  font-size: 0.8125rem;
+  line-height: 138%;
+  display: inline-block;
+  padding: 0.3125rem 0;
+  transition: 0.3s;
+  width: 100%; /* Ensuring anchor tags take full width of container */
+  max-width: 100%
+}
+
+.footer-disclaimer a {
+  color: #e87817;
+  text-decoration: none;
+  background-color: transparent;
+  font-size: 0.6875rem;
+  transition: 0.3s;
+}
+
+.footer-links a:hover {
+  color: var(--color-white);
+}
+
+@media (width > 48rem) {
+  .footer-links .links-column{
+    flex: 0 0 auto;
+    width: 16.6667%;
+  }
+}
+
+.footer-links .links-column > div{
+  margin: 0 0 1.25rem
+}
+
+@media (width < 37.5rem) {
+  .footer-top-note p {
+    padding-left: 0.9375rem;
+    padding-right: 0.9375rem;
+  }
+}
+
+.footer-top .row {
+  display: flex;
+  flex-wrap: wrap;
+  margin-right: -0.9rem;
+  margin-left: -0.9rem;
+}
+
+.footer-links .row {
+  display: flex;
+  flex-wrap: wrap;
+  margin-right: -0.9rem;
+  margin-left: -0.9rem;
+}
+
+@media (width >= 56.25rem) {
+  .section > .section-container {
+    max-width: 81.25rem;
+    margin: auto;
+  }
+}
+
+.footer-disclaimer {
+  background: #3d3d3d;
+  padding-top: 1.875rem;
+}
+
+.footer-disclaimer a:hover {
+  color: var(--link-color-blue);
+}
+
+.footer-disclaimer .panel-header {
+  position: relative;
+  padding: 0.625rem 0;
+  padding-right: 2.5rem;
+  z-index: 4;
+  cursor: pointer;
+}
+
+.footer-disclaimer .panel-header strong {
+  font-size: 0.9375rem;
```

Comment:
please use constants in style.css for font sizes

Reply:
I have opted size from style.css but some are odd numbers (11.13..) which need to maintain same look.

Reply:
I think in that case nearest font size 11px should be picked that would help in keeping the layout relatively uniform

Reply:
sorry for typo i was meant to write 11, 13 etc..

Reply:
used global one.